src/components/AccordionPanel/AccordionPanelItem.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+
+type Props = {
+  children: React.ReactNode
+  name: string
+  className?: string
+}
+
+type ContextType = Omit<Props, 'children'>
+
+export const AccordionPanelItemContext = React.createContext<ContextType>({
+  name: '',
+})
+
+export const AccordionPanelItem: React.FC<Props> = ({ children, name, className = '' }) => {
+  return (
+    <AccordionPanelItemContext.Provider
+      value={{
+        name,
+      }}
+    >
+      <div className={className}> {children}</div>
```

Comment:
[ask] Why do you put a space in front of `{children}`?

Reply:
This is just typo